xwalk_android.gypi

```diff
@@ -213,6 +213,59 @@
       'includes': ['../build/jni_generator.gypi'],
     },
     {
+      'target_name': 'xwalk_runtime_lib_lzma_compress',
+      'type': 'none',
+      'dependencies': [
+        'xwalk_runtime_lib_apk',
+      ],
+      'variables': {
+        'files_to_compress': [
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/stripped_libraries/libxwalkcore.so',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/classes.dex',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/icudtl.dat',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/xwalk.pak',
+        ],
+        'assets_dir': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/assets/',
+      },
+      'actions': [
+        {
+          'action_name': 'lzma_compress',
+          'message': 'compress ingredients for runtime APK',
+          'inputs': [
+            'build/android/lzma_compress.py',
+          ],
+          'outputs': [
```

Comment:
I still think that instead of hardcoding the list here you could add a mode to the new Python script to output those names. This would also allow you to do more in `files_to_compress`, like maybe putting the V8 snapshot files there in the future if/when we start using them.